akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -78,9 +78,6 @@ it is recommended to upgrade the oldest node last. This way cluster singletons a
 Otherwise, in the worst case cluster singletons may be migrated from node to node which requires coordination and initialization 
 overhead several times.
 
-[Kubernetes Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) with `RollingUpdate`
-strategy will roll out updates in this preferred order, from newest to oldest. 
-
```

Comment:
I think you are mixing up leadership choice (ip sort order) with oldest node used by cluster singleton which is in fact the oldest node to have joined the cluster. (https://github.com/akka/akka/blob/3a96f06e3f8ddaa6f850a343324e709847ec316f/akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala#L286)

Reply:
oh, yes. Sorry about that and thank you for showing me the code the code  👍

Reply:
Singletons (Sharding Coordinator) runs on the oldest, and that is the order the Akka Cluster members were moved to Up status.

There is one caveat that we don't mention in the docs (and I don't think we have to). That is when starting the Akka Cluster with several pods at the same time. The Akka oldest might not be exactly the same as the Kubernetes oldest. That sorts itself out after the first rolling update.

When starting several pods at the same time the one with lowest address will join itself and become the Akka oldest. It's immediately moved to status Up. That might not be the same as the oldest pod start timestamp.

Reply:
Even more insights. This is great.
So the first time we deploy the cluster this can happen and also if we do a Rolling Update allowing multiple pods starting at once with 
```
rollingUpdate:
      maxSurge: [x]
```
where `[x] > 1`

Understood I'll keep that in mind. Thanks @patriknw

Reply:
Yes, that is right. one-by-one is best if possible.